src/constants/response.constant.ts

```diff
@@ -0,0 +1,6 @@
+export const RESPONSE = {
+  FACILITY_TERM_CREATED: 'Facility Term created',
+  FACILITY_TERM_NOT_CREATED: 'Facility Term not created',
+  FACILITY_TERM_EXISTS: 'Facility Term already exists',
+  INTERNAL_SERVER_ERROR: 'Internal Server Error',
```

Comment:
These 2 values are only used by tests so they might be better in the `test/support` folder

Reply:
Moved constants to `test/support`